frontend/src/components/Item/Card/Card.vue

```diff
@@ -83,176 +85,134 @@ import {
-      if (this.item.Type === 'Episode' && this.item.SeriesId) {
-        return this.getItemDetailsLink({ Id: this.item.SeriesId }, 'Series');
-      }
+        if (props.item.ProductionYear?.toString() === endYear) {
+          return props.item.ProductionYear.toString();
+        }
 
-      return this.getItemDetailsLink(this.item);
-    },
-    /**
-     * Gets a link to be applied to the card subtitle
-     *
-     * @returns A router link to the parent item or a related item
-     */
-    cardSubtitleLink(): string | undefined {
-      if (
-        this.item.Type === 'MusicAlbum' &&
-        this.item.AlbumArtists &&
-        this.item.AlbumArtists.length > 0
-      ) {
-        return this.getItemDetailsLink(
-          this.item.AlbumArtists[0],
-          'MusicArtist'
-        );
-      } else if (this.item.Type === 'Episode') {
-        return this.getItemDetailsLink(this.item);
+        return `${props.item.ProductionYear} - ${endYear}`;
       }
 
-      return undefined;
-    },
-    progress(): number | false {
-      return this.item.UserData?.PlayedPercentage || false;
-    },
-    getImageType(): ImageType {
-      return this.shape === CardShapes.Thumb
-        ? ImageType.Thumb
-        : ImageType.Primary;
-    },
-    /**
-     * Gets the library update progress
-     */
-    refreshProgress(): number | undefined {
-      return this.taskManager.getTask(this.item.Id || '')?.progress;
+      break;
+    }
+    default: {
+      return `${props.item.ProductionYear || ''}`;
     }
-  },
-  methods: {
-    isFinePointer(): boolean {
-      return window.matchMedia('(pointer:fine)').matches;
-    },
-    getItemDetailsLink,
-    canPlay
   }
+
+  return '';
 });
+
+/**
+ * Gets a link to be applied to the card title
+ *
+ * @returns A router link to the item or a related item
+ */
+const cardTitleLink = computed(() => {
+  if (props.item.Type === 'Episode' && props.item.SeriesId) {
+    return getItemDetailsLink({ Id: props.item.SeriesId }, 'Series');
+  }
+
+  return getItemDetailsLink(props.item);
+});
+
+/**
+ * Gets a link to be applied to the card subtitle
+ *
+ * @returns A router link to the parent item or a related item
+ */
+const cardSubtitleLink = computed(() => {
+  if (
+    props.item.Type === 'MusicAlbum' &&
+    props.item.AlbumArtists &&
+    props.item.AlbumArtists.length > 0
+  ) {
+    return getItemDetailsLink(props.item.AlbumArtists[0], 'MusicArtist');
+  } else if (props.item.Type === 'Episode') {
+    return getItemDetailsLink(props.item);
+  }
+});
+
+const progress = computed(
+  () => props.item.UserData?.PlayedPercentage || undefined
+);
+
+const getImageType = computed(() =>
+  props.shape === CardShapes.Thumb ? ImageType.Thumb : ImageType.Primary
+);
+
+/**
+ * Gets the library update progress
+ */
+const refreshProgress = computed(
+  () => taskManager.getTask(props.item.Id || '')?.progress
+);
+
+const isFinePointer = computed(
+  () => window.matchMedia('(pointer:fine)').matches
```

Comment:
Not sure this is reactive, to be sure maybe we can switch to https://vueuse.org/core/usemediaquery/#usemediaquery ?

Reply:
Used useMediaQuery